pkg/apis/migration/v1alpha1/directvolumemigration_types.go

```diff
@@ -33,8 +33,14 @@ type PVCToMigrate struct {
 type DirectVolumeMigrationSpec struct {
 	SrcMigClusterRef            *kapi.ObjectReference `json:"srcMigClusterRef,omitempty"`
 	DestMigClusterRef           *kapi.ObjectReference `json:"destMigClusterRef,omitempty"`
+
+	//  A list of all the PVCs that are to be migrated with direct volume migration
```

Comment:
The type is already displayed and re-stating it in the comment might not be necessary.

example:
```
   paused       <boolean>
   Indicates that the deployment is paused.
```